src/styles/partials/components/_emphasized-image.scss

```
@@ -55,8 +55,11 @@
 }
 
 @media screen and (max-width: $small-breakpoint) {
-  p:first-of-type,
+  p:first-of-type {
+    margin: $base-spacing-unit * 3 auto;
+  }
   p:last-of-type {
+    margin-bottom: $base-spacing-unit * 9, auto;
```

comma here should throw an error when you build the style sheet.